### PR TITLE
Feature/js scripts refactor

### PR DESCRIPTION
**Pull request by kendallha**
# Describe Changes

Please include changes made or which issues were fixed.

# Type of Change
- [ ] Feature HTML
- [ ] Feature JavaScript
- [ ] Feature CSS
- [x ] Refactor/Fix Bugs

# Please describe the change made

Please give a short description of the changes that were made below
- Fix:
Refactored scripts file to instantiate class instances globally and use class properties and methods where appropriate. Refactored functions in the scripts file to reduce unnecessary code and logic. 

# How has this been tested?

- [x ] In Browser
- [ ] In Console

### CODE REVIEW

File: src/dom-updates.js [hunk 1]
```diff
@@ -8,7 +8,43 @@ let domUpdates = {
...
+ },
+
+ createCards(recipeRepo, element) {
+  // element.innerHTML = ` <div id='recipe-instructions' class="recipe-instructions">
+  //     </div>
+  //     <div class="my-recipes-banner">
+  //       <h1>My Recipes</h1>
+  //       <button class="show-all-btn">Show All Recipes</button>
+  //     </div>`;
+  // this.removeElements(element);
```
Comment (MarcelineBall) [inline src/dom-updates.js:32]: If we don't end up using this, we should probably delete it when we're done with all the refactoring

File: src/dom-updates.js [hunk 2]
```diff
@@ -40,10 +76,12 @@ let domUpdates = {
   },
 
   hideUnselectedRecipes(foundRecipes) {
-    foundRecipes.forEach(recipe => {
-      let domRecipe = document.getElementById(`${recipe.id}`);
+    // foundRecipes.forEach(recipe => {
```
Comment (errabun) [inline src/dom-updates.js:79]: Let's be sure to revisit these commented out sections

**Review by errabun — COMMENTED** [review]
great work bringing refactoring all those DOM functions that were in scripts.js!

**Review by MarcelineBall — APPROVED** [review]
This looks great! Good job moving the functions to the dom updates and calling them from there. We may want to delete some of the commented out sections later on if they don't see use in the final product

**Pull request merged**